Floofbot/Services/EventLoggerService.cs

```diff
@@ -389,11 +389,18 @@ public Task UserLeft(IGuildUser user)
                         return;
 
                     var embed = new EmbedBuilder();
-
                     embed.WithTitle($"❌ User Left | {user.Username}#{user.Discriminator}")
                          .WithColor(Color.Red)
-                         .WithDescription($"{user.Mention} | ``{user.Id}``")
-                         .WithFooter($"user_leave user_joinlog {user.Id}")
+                         .WithDescription($"{user.Mention} | ``{user.Id}``");
+                    if (user.JoinedAt != null)
+                    {
+                        DateTimeOffset userJoined = ((DateTimeOffset)user.JoinedAt);
+                        TimeSpan interval = DateTime.UtcNow - userJoined.DateTime;
+                        string day_word = interval.Days == 1 ? "day" : "days";
+                        embed.AddField("Joined Server", userJoined.ToString("ddd, dd MMM yyyy"), true);
+                        embed.AddField("Time at Server", $"{interval.Days} {day_word}", true);
```

Comment:
"Time at Server" sounds like "current local time on the server". I wonder if we can come up with another description for this field, such as "Amount of Time in Server", or if we're sticking to just days, "# Days in Server"?
I do think using yrs/months would help too, although I think it depends on what we're doing.
For example, if we're going to scrape over all of the leave times then using just days is better.

Reply:
From the context and valua, it should be easily seen it's not the local time of the server. Though maybe "Member for" or something like that could be better.